install.rdf

```diff
@@ -26,8 +26,9 @@
     <em:description>js-ctypes wrapper for libotr</em:description>
     <em:homepageURL>https://github.com/arlolra/ctypes-otr</em:homepageURL>
     <em:optionsURL>chrome://otr/content/prefs.xul</em:optionsURL>
-    <em:updateURL>https://127.0.0.1/</em:updateURL>
+    <em:updateURL>data:text/plain,</em:updateURL>
+    <em:updateKey>-</em:updateKey>
```

Comment:
@azadi is getting these lines from,

https://gitweb.torproject.org/tor-launcher.git/commit/?id=06f135535fac7e79be9051229030d195a44c84a7

I think omitting them results in dialling home, looking for a newer version, which we're trying to prevent.  But I don't remember the exact reasoning.